### PR TITLE
pkp/pkp-lib#7134 Show relation status when preprint is posted

### PR DESCRIPTION
**Pull request by ajnyga**
https://github.com/pkp/pkp-lib/issues/7134

### CODE REVIEW

**Comment by ajnyga** [comment]
Just to make sure I understand.

Do you want these to relation form:

```
msgid "publication.relation.none"
msgstr "Preprint has not been submitted for publication."

msgid "publication.relation.submitted"
msgstr "Preprint has been submitted for publication in a journal or other academic publication."

msgid "publication.relation.published"
msgstr "Preprint has been published in a journal or other academic publication."
```

And these different strings to the `publishForm`:

```
msgid "publication.publish.relationStatus.none"
msgstr "This preprint has not been published elsewhere."

msgid "publication.publish.relationStatus.submitted"
msgstr "This preprint has been submitted to a journal or other academic publication."

msgid "publication.publish.relationStatus.published.noDoi"
msgstr "This preprint has been published, but no DOI is available yet."

msgid "publication.publish.relationStatus.published"
msgstr "This preprint has been <a href='{$vorDoi}' target='_blank'>published</a>."
```

**Comment by ajnyga** [comment]
also, is this something that we should include in tests? 

- Post a preprint with no relation (check status in publishForm)
- Create a new version with published+doi status and post that online (check status in publishForm)

File: classes/components/forms/publication/PublishForm.inc.php [hunk 1]
```diff
@@ -70,13 +70,33 @@ public function __construct($action, $publication, $submissionContext, $requirem
             ]);
         }
 
+        // Related Publication status
+        if ($publication->getData('relationStatus') == \Publication::PUBLICATION_RELATION_PUBLISHED && $publication->getData('vorDoi')) {
+            $relationStatus = __('publication.relation.published');
+            $relationStatus .= '<br /><a target="_blank" href="' . $publication->getData('vorDoi') . '">' . $publication->getData('vorDoi') . '</a>';
```
[the remark this answers is not in the document]
Comment (ajnyga) [reply classes/components/forms/publication/PublishForm.inc.php:76]: oops, fixed

File: classes/components/forms/publication/PublishForm.inc.php [hunk 2]
```diff
@@ -70,13 +70,33 @@ public function __construct($action, $publication, $submissionContext, $requirem
...
+        } elseif ($publication->getData('relationStatus') == \Publication::PUBLICATION_RELATION_SUBMITTED) {
+            $relationStatus = __('publication.relation.submitted');
+        } else {
+            $relationStatus = __('publication.relation.none');
```
[the remark this answers is not in the document]
Comment (ajnyga) [reply classes/components/forms/publication/PublishForm.inc.php]: I was unsure if you want separate strings for these that you have in the above post. Anyhow, I updated the pr with totally new strings. Happy to go back if needed.

**Comment by NateWr** [comment]
Thanks this looks good to me. If you're happy with it I can merge this in, but wanted to double check since it's been a few weeks.

**Comment by ajnyga** [comment]
@NateWr go ahead and merge

**Comment by asmecher** [comment]
(Merging on Nate's behalf, thanks @ajnyga!)

**Pull request merged**